src/backend/src/utils/response.utils.ts

```diff
@@ -0,0 +1,19 @@
+import { Response } from 'express';
+
+/**
+ * Send a success json response
+ * @param res the Express response object
+ * @param obj any object we want to send back (e.g., User, Project, Risk[], number)
+ */
+export const sendSuccessJsonResponse = (res: Response, obj: any) => {
```

Comment:
I don't see the value in this method.

I feel like we'd want to control the type of success code, 201 for created, 204 for no-content on delete, 200 for Ok, most get responses

Reply:
good point, i'll add the code.

the reason i want these methods though is more for the sendSuccessMessageResponse response, because i want to make it easy that we always send {message: 'message'}